packages/formspree-react/src/useForm.ts

```diff
@@ -171,6 +168,7 @@ const useForm = (
             : undefined,
       })
       .then((result: SubmissionResponse) => {
+        // @ts-ignore: unhandled result.response is possibly null
```

Comment:
I tried to fix as many type errors without changing the runtime behavior. It's impossible to fix these without changing the implementation to handle `null` better. We'll do in a follow-up PR.